README.md

```diff
@@ -268,4 +280,11 @@ You can use pytest to run tests against your current Python version. To run test
 See [`setup.py`](setup.py) for test dependencies and install them with .[test].
 
 ## Contributors
-sloev, svdgraaf, simongarnier, elviejokike, paddie, AlbertoTrinidade, martinbuberl
+
+@nabrosimoff, @elviejokike, @eduardomourar, @devgrok, @AlbertoTrindade, @paddie, @svdgraaf, @simongarnier, @martinbuberl, @sloev
+
+**Wanna contribute?**
+
+* crete issues or PR's, no real guidelines given here.
+* @sloev will review and take care of version bumping and release etc
```

Comment:
I wouldn't mention myself here, just write a contributed will look at it.

Reply:
yeah, good idea!